### ISSUE TITLE
Full support for material-components-web ?

### ISSUE THREAD

**Issue by felixakiragreen** [kradio3/react-mdc-web#7]
This library looks really promising so far. Definitely the best react implementation of `material-components-web` I could find. I would like to use this library for a production web app. Are you planning on fully implementing and supporting all the m-c-w components? Any idea how long you will be supporting it for? (As an open-source creator myself, I don't want to assume you're fully committed to this project!)

Looking through your project, I see it's almost complete, just a few are missing:
- [ ] mdc-animation 
- [ ] mdc-auto-init
- [ ] mdc-base  
- [x] mdc-button (missing ripple)
- [x] mdc-card
- [x] mdc-checkbox  
- [x] mdc-drawer  
- [x] mdc-dialog
- [x] mdc-elevation 
- [x] mdc-fab 
- [x] mdc-form-field  
- [x] mdc-grid-list
- [x] mdc-icon-toggle 
- [x] mdc-layout-grid 
- [x] mdc-list  
- [x] mdc-menu  
- [x] mdc-radio 
- [ ] mdc-ripple  
- [ ] mdc-rtl 
- [ ] mdc-select  
- [x] mdc-snackbar  
- [x] mdc-switch  
- [x] mdc-textfield (missing advanced states)
- [ ] mdc-theme 
- [x] mdc-toolbar 
- [x] mdc-typography

**Comment by kradio3** [comment]
@dubert , thank you for your appreciation. Sure, I want to implement all of them and no plans to stop supporting. At the moment i'm implementing Menu component and extending Textfield for fullwidth and multiline. Keep in touch :+1:

**Comment by felixakiragreen** [comment]
Do you have any starting points where you would appreciate contribution?

**Comment by kradio3** [comment]
I've created a few issues from your list. They marked as 'enhancement'. You can assign one of them or create a new one which you would like. Be careful, i've already assigned #13 :)

**Comment by demiters** [comment]
Just wanted to say I appreciate what you guys are doing! I'll be using this in production also.

**Comment by pricetula** [comment]
Very very nice project....is it possible to have stuff like selects and maybe auto complete. Great guys

**Comment by kradio3** [comment]
@pricetula Yes, sure. I just want to finish Menu component. A little bit tricky there

**Comment by sayjeyhi** [comment]
very good work, it will be a great library after finishing all (specially RTL support)
